tests/bin/test-integration-ec2.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+#
+# Preps a test environment and runs `make test-integration`
+# against artifacts produced from the current source tree
+#
+
+# fail on any command exiting non-zero
+# set -eo pipefail
+set -eo
```

Comment:
No, `pipefail` won't work because of [this line](https://github.com/mboersma/deis/commit/57ca0c0f5a12fdf629a806ce87414d57956711f3#diff-5b34da9043b2bea3c75b7e5dd653b036R73). I'm open to suggestions.